tests/phpunit/WC_Stripe_Webhook_Handler_Test.php

```
@@ -296,6 +296,91 @@ function ( $passed_order ) use ( $order ) {
+		$notification = (object) [
+			'type' => 'payment_intent.succeeded',
+			'data' => (object) [
+				'object' => (object) self::MOCK_PAYMENT_INTENT,
+			],
+		];
+
+		$this->mock_webhook_handler->expects( $this->never() )
+			->method( 'handle_deferred_payment_intent_succeeded' );
+
+		$this->mock_webhook_handler->process_deferred_webhook( 'payment_intent.succeeded', $data, $notification );
+	}
```

PHPMD (2.15.0)

[error] The method test_deferred_webhook_skipped_when_order_already_paid is not named in camelCase. (undefined)

(CamelCaseMethodName)